deploy/example_catalog/cr-app-datafabric610.json

```diff
@@ -0,0 +1,232 @@
+	    "zookeeper",
+            "cldb"
+          ]
+        },
+        {
+        "roleID": "oozie",
+        "serviceIDs": ["warden", "oozie", "mysql"]
+      },
+        {
+          "roleID": "edge",
+          "serviceIDs": [
+            "mapr-client",
+            "ssh"
+          ]
+        },
+        {
+          "roleID": "nodemanager",
+          "serviceIDs": [
+            "warden",
+            "yarn-nm"
+          ]
+        },
+        {
+          "roleID": "resource-manager",
+          "serviceIDs": [
+            "warden",
+	    "zookeeper",
+            "yarn-rm"
+          ]
+        }
+      ],
+      "selectedRoles": [
+        "control-system",
+        "resource-manager",
+        "cldb",
+        "oozie",
+        "edge",
+        "nodemanager"
+      ]
+    },
+    "label": {
+      "name": "Ezmeral Datafabric 610",
+      "description": "Ezmeral Datafabric 610 with YARN and Oozie"
+    },
+    "distroID": "ezmeral/datafabric610",
+    "version": "1.0",
+    "configSchemaVersion": 7,
+    "services": [
+      {
+        "id": "mapr-cs",
+        "label": {
+          "name": "Datafabric Control System"
+        },
+        "endpoint": {
+          "urlScheme": "https",
+          "path": "/app/mcs",
+          "port": 8443,
+          "isDashboard": true
+        }
+      },
+      {
+        "id": "yarn-rm",
+        "label": {
+          "name": "ResourceManager"
+        },
+        "endpoint": {
+          "urlScheme": "https",
+          "path": "/cluster",
+          "port": 8090,
+          "isDashboard": true
+        }
+      },
+      {
+        "id": "yarn-nm",
+        "label": {
+          "name": "NodeManager"
+        },
+        "endpoint": {
+          "urlScheme": "https",
+          "path": "/node",
+          "port": 8044,
+          "isDashboard": true
+        }
+      },
+      {
+        "id": "yarn-hs",
+        "label": {
+          "name": "YARN HistoryServer"
+        },
+        "endpoint": {
+          "urlScheme": "https",
+          "path": "/jobhistory",
+          "port": 19890,
+          "isDashboard": true
+        }
+      },
+      {
+        "id": "cldb",
+        "label": {
+          "name": "CLDB Web Port"
+        },
+        "endpoint": {
+          "port": 7443,
+          "isDashboard": false
+        }
+      },
+      {
+        "id": "mapr-client",
+        "label": {
+          "name": "Datafabric Client."
+        }
+      },
+      {
+        "id": "zookeeper",
+        "label": {
+          "name": "Zookeeper Server"
+        },
+        "endpoint": {
+          "port": 5181,
+          "isDashboard": false
+        }
+      },
+      {
+      "id": "oozie",
+      "label": {
+        "name": "OOZIE"
+      },
+      "endpoint": {
+        "urlScheme": "https",
+        "path": "/oozie",
+        "port": 11443,
+        "isDashboard": true
+      }
+    },
+      {
+        "id": "mysql",
+        "label": {
+          "name": "MySQL"
+        }
+      },
```

Comment:
no need to expose this?

Reply:
MySQL is for Oozie and so i am not exposing it.